README.md

```diff
@@ -22,212 +18,22 @@ Tabour is a Kotlin library which makes working with queues/topics much easier.
-```
-
-### Consumer example
-
-For consumption, you can extend the ProtoEventConsumer, which implements a helper function deserializing JSON to your
-protobuf as such
-
-```kotlin
-class BookingEventConsumer : ProtoEventConsumer<BookingEvent.Builder>() {
-
-    override fun consume(message: String) {
-        val event = parseMessageToEvent(message, BookingEvent.newBuilder()).build()
-        logger.debug { "consumed event: $event" }
-        // ... perform business logic on your proto event ...
-    }
-}
-```
-
-## Configurations
-
-<table>
-<thead><tr><th>path</th><th>default value</th><th>explanation</th></tr></thead>
-<tr>
-<td><pre>tabour.max-retry-count</pre></td>
-<td><pre>3</pre></td>
-<td>Number of times that is going to retry to publish  or consume an event</td>
-</tr>
-<tr>
-<td><pre>tabour.poller.poll-delay</pre></td>
-<td><pre>NA</pre></td>
-<td>Delay the poller should wait for the next poll after the previous poll has finished</td>
-</tr>
-<tr>
-<td><pre>tabour.poller.wait-time</pre></td>
-<td><pre>20 Seconds</pre></td>
-<td>The duration (in seconds) for which the call waits for a message to arrive in the queue before returning. If a message is available, the call returns sooner than WaitTimeSeconds. If no messages are available and the wait time expires, the call returns successfully with an empty list of messages.</td>
-</tr>
-<tr>
-<td><pre>tabour.poller.visibility-timeout</pre></td>
-<td><pre>360 Seconds</pre></td>
-<td>Visibility timeout is the time-period or duration you specify for the queue item which when is fetched and processed by the consumer is made hidden from the queue and other consumers.</td>
-</tr>
-<tr>
-<td><pre>tabour.poller.batch-size</pre></td>
-<td><pre>10</pre></td>
-<td>The maximum number of messages to pull from the even bus each poll:
-
-     *  event bus:
-        - SQS allows is maximum 10</td>
-
-</tr>
-<tr>
-<td><pre>tabour.poller.num-of-pollers</pre></td>
-<td><pre>10</pre></td>
-<td>The number of coroutines that should poll/process new messages. Each of those coroutines will poll a batch of batchSize messages and then wait for the pollDelay interval until polling the next batch.</td>
-</tr>
-<tr>
-<td><pre>tabour.sqs.access-key</pre></td>
-<td><pre>NA</pre></td>
-<td>The AWS access key.</td>
-</tr>
-<tr>
-<td><pre>tabour.sqs.secret-key</pre></td>
-<td><pre>NA</pre></td>
-<td>The AWS secret key.</td>
-</tr>
-<tr>
-<td><pre>tabour.sqs.region</pre></td>
-<td><pre>NA</pre></td>
-<td>The AWS region</td>
-</tr>
-<tr>
-<td><pre>tabour.sqs.enable-consumption</pre></td>
-<td><pre>false</pre></td>
-<td>Configures if this the sqs listeners should be starting</td>
-</tr>
-</table>
+The library contains the following modules
+
+- [Core](core/README.md)
+- Proto
+- Spring
+
+To use Tabour only **core** is required. Proto and Spring packages contain helpers and functionality regarding 
+- Protobuf
+- Spring Boot
+
+
+## Supported messaging systems
+
+- SQS
 
```

Comment:
Let's use better linkage for the docs here. An ordering about what to read first etc will be helpful